apps/modernization-ui/src/components/Table/table.module.scss

```diff
@@ -17,6 +17,9 @@
                     border: none;
                     align-items: center;
                 }
+                button {
+                    padding: 0;
+                }
                 button:disabled {
                     cursor: text;
```

Comment:
Just adding here caused this bug when hovering 
![Screenshot 2024-02-05 at 1 06 52 PM](https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/59c68b78-0fbf-4949-8206-4f8c36fb35c9)